static/src/stylesheets/module/facia/snaps/_football.scss

```diff
@@ -257,3 +256,8 @@ $footballBadgeSizeDesktop: 120px;
     width: 100%;
     padding: $gs-baseline/1.5;
 }
+
+.football-matches__group-name {
+    float: right;
+    padding: .5rem;
```

Comment:
might want to use `em` here, in case those font sizes change? `rem` will always be the same, even if they do